cmd/scatr/version.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+const version = "0.2.1"
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Returns the current SCATR version",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("SCATR analyzer testing framework")
+		fmt.Println("  Version:", version)
```

Comment:
nitpick though not required.

You can use (similar):
```
fmt.Printf("%2s: %s\n", "Version", version)
```

or https://pkg.go.dev/text/tabwriter

But for such a small thing, I do not think it is required!